piff/gp_interp.py

```diff
@@ -173,42 +216,84 @@ def interpolateList(self, stars, logger=None):
         :returns: a list of new Star instances with interpolated parameters
         """
         Xstar = np.array([self.getProperties(star) for star in stars])
-        y = self._predict(Xstar)
+        gp_y = self._predict(Xstar)
         fitted_stars = []
-        for y0, star in zip(y, stars):
+        for y0, star in zip(gp_y, stars):
             if star.fit is None:
```

Comment:
This branch is not covered.  Is this possible to reach?  Not sure what test to add to hit this line.

Reply:
I added a test for that now